rocketmq-flink/src/main/java/org/apache/rocketmq/flink/RocketMQSource.java

```diff
@@ -84,6 +85,7 @@
     private static final String OFFSETS_STATE_NAME = "topic-partition-offset-states";
 
     private transient volatile boolean restored;
+    private boolean enableCheckpoint;
```

Comment:
transient variable.

Reply:
no need to set this variable as transient

Reply:
> no need to set this variable as transient

:+1: